tests/problem1Tests.rb

```diff
@@ -0,0 +1,37 @@
+require 'test/unit'
+require '../src/problem1'
+
+class Problem1Tests < Test::Unit::TestCase
+
+  def setup
+    @problem1 = Problem1.new
+    @randomIntegerTestData = [1,4,7,4,10,6]
+    @orderedIntegerTestData = (1..100).to_a
```

Comment:
I suggest following a coding convention for any programming language.

Ruby: https://github.com/bbatsov/ruby-style-guide